### ISSUE TITLE
Windows ?

### ISSUE THREAD

**Issue by gedw99** [tigrisdata/tigris#634]
https://github.com/tigrisdata/tigris/blob/main/scripts/install_build_deps.sh seems to not support windows.

is that because foundation db just does not run on windows ?

**Comment by efirs** [comment]
You are right, install_build_deps.sh needs to be fixed to support Windows.

But most of the development flows don't require FoundationDB clients on the host.
So if you disable platform check in the script and install the rest of the tools, you
would be able to run:

* `make test` which would execute the tests in the docker container.
* `make generate` to generate the code, so all symbols are resolved in the IDE.
* `make run` to bring up a Tigris instance on localhost:8081, so you can issue manual requests.

I did simple manual tests on Windows in the past, but it could be broken since then.

We would greatly appreciate the contributions.
